### ISSUE TITLE
Errors installing BeautifulSoup

### ISSUE THREAD

**Issue by dnigendac** [rationalmatter/Juno-Issue-Tracker#136]
**Describe the bug**
Can't install beautifulsoup4

Seems like an issue with [how BeautifulSoup4 detects Python 3](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#problems-after-installation).

However installing BeautifulSoup4 on Mac (`pip3 install beautifulsoup4`) and manually copying the `bs4` directory to Juno's `site-packages` works without issues.

**To Reproduce**
1. Install BeautifulSoup4 with the included package installer
2. Run `from bs4 import BeautifulSoup`
3. See error

**Expected behavior**
Expected no error.

**Screenshots**
Cell output:
```

Traceback (most recent call last):

  File "/var/mobile/Containers/Data/Application/.../Library/Application Support/com.rationalmatter.junoapp/python-home/lib/python3.6/site-packages/IPython/core/interactiveshell.py", line 3291, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-1-f33cf0cb8fd3>", line 1, in <module>
    from bs4 import BeautifulSoup

  File "/var/mobile/Containers/Data/Application/.../Documents/site-packages/bs4/__init__.py", line 49
    'You are trying to run the Python 2 version of Beautiful Soup under Python 3. This will not work.'<>'You need to convert the code, either by installing it (`python setup.py install`) or by running 2to3 (`2to3 -w bs4`).'
                                                                                                       ^
SyntaxError: invalid syntax
```

**Environment (please complete the following information):**
 - Device: iPad Pro 11"
 - iOS version: 13.3.1
 - App version 1.1.1

**Comment by navoshta** [comment]
Thanks for reporting @dnigendac — this is a bug in Juno, I'm afraid. It grabs the first available Python-only source-only wheel instead of trying to find the Python 3 one. This is fixed now, and will be released in Juno 1.2. I can let you know when Juno beta becomes available, in case you would like to help test it. 😉

**Comment by dnigendac** [comment]
Great to know, thanks! Would be happy to test 😄

**Comment by navoshta** [comment]
Juno 1.2 is now waiting for TestFlight review, and should be available in the next few days — please send an e-mail to beta@juno.sh if you would like to help test it! It should fix a few issues around PyPI installer.

**Comment by dnigendac** [comment]
Thanks for adding me, looking good to me on 1.2!

**Comment by navoshta** [comment]
Great, thanks for verifying! I'll close the issue when 1.2 is released in the App Store.

**Comment by Axcyn** [comment]
Hello,
I reopen the ticket because with the latest version of Juno 1.3.9 I can’t install Beautiful Soup on my iPad Pro 12.9" with iPadOS 14.6.

I’m not a regular user of Juno nor an advanced user in Python so maybe I did something wrong. Below is what I did :

When I tried to install the package using the built-in installation manager I got the following error : `Failed to install bs4 installation from source failed`.

So I downloaded the package directly from the website : `beautifulsoup4-4.9.3-py3-none-any` and put it directly into `site-packages` and I can’t use it.

```---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-d99746be2408> in <module>
----> 1 from bs4 import BeautifulSoup

ModuleNotFoundError: No module named 'bs4'
